client/state/utils/README.md

```diff
@@ -0,0 +1,18 @@
+Utils
+========
+
+A module for those little helpers that need to interact woth the store.
+
+# Actions
+Used in combination with the Redux store instance `dispatch` function, actions can be used in manipulating the current global state.
+
+## uniqueId( prefix )
+Meant as a replacement for _ and lodash "uniqueId" function.
+Returns unique numeric prefixed by prefix and increments internal state, so the next uniqueId will be different.
```

Comment:
Action creators should return actions. Still thinking about how else we might do this...